### ISSUE TITLE
Phase 1 Close-Out Checklist

### ISSUE THREAD

**Issue by mbarnett** [ualbertalib/jupiter#649]
Finishing out Phase 1
---------------------
- [ ] #702 – PMPY Jupiter-side
- [x] #529 – Embargo expiry job in place
- [x] #471 – Upgrade to impending Rails 5.2 release (Update: [out now!](http://weblog.rubyonrails.org/2018/4/9/Rails-5-2-0-final/)), move off of our hacked-up version of ActiveStorage 0.1 
    - [ ] Related/Blocks: (no issue) – download files w/o Fedora's involvement
    - [ ] Related/Blocks: #435 – serve thumbnails w/o Rails involvement
    - [ ] Related/Blocks: #362 – allow users to order/reorder files in upload
- [x] #709 - Thesis Deposit by admin users
- [ ] #693 - Default Search results sorted by relevance. 
- [x] #224 – Everything done on the Go Live Checklist
- [ ] #529 – map user IDs to email addresses for use in Metadata team's triplestore (postpone to after our review work on metadata/object repository done)
- [x] #650 – Remove confusing counts from inactive search tabs
- [x] #551 – Download & the back button/ log-in redirection isn't playing well together
- [ ] #550 – Don't attempt to generate thumbnails for particularly large items
- [x] #410 – Script to regenerate Solr data from Fedora directly, for recovery scenarios in which Fedora and Solr backups are not perfectly in sync (likely)
- [x] #656 – Bot Hardening
- [x] #655 – Remove description validation on Drafts
- [x] #670 – 503 OAI paths
- [x] #664 – sitemap fixes
- [x] #704 – Monitor Google Search Console results
- [x] (no issue) – staging server not currently useful for pre-prod smoke testing (missing most data, no thumbnails, can't log in due to IST setup missing, etc, etc)
- [x] #512 – Allow admins to relocate collections to a different community

Unsure
--------
- [ ] (no issue) – Background Fedora ingestion for better self-deposit performance (Blocked by #471)
- [ ] #387 – Allow for multiple non-admin users to posses permissions to edit an object

Post-Phase 1 But Needs to Happen Pre-Phase 2/Digitization
-------------------------------------------------------------
- [ ] (no issue) – Figure out long-term role of Fedora/ActiveFedora (if any), spike out removal of ActiveFedora usage from JupiterCore/transition to RDBMS for driving the web app, as performance for large digitized items will be unworkable in current setup (1000 page books, say).
    - [ ] Related: #630 – Solrizer, which we rely on,  is deprecated. Need a plan to move away from it before any critical bugs/security issues put us in a bad spot.

**Comment by weiweishi** [comment]
We've completed phase 1. Close this ticket.
